src/graphforge/datasets/loaders/compression.py

```
@@ -4,22 +4,67 @@
+
+def _validate_archive_member(name: str) -> None:
+    """Validate an archive member name for security (absolute or traversal).
+
+    Args:
+        name: Archive member name to validate
+
+    Raises:
+        ValueError: If the name is unsafe (absolute path or contains parent traversal)
+    """
+    # Debug on Windows
+    import sys
```

Ruff (0.14.14)

[error] Redefinition of unused `sys` from line 15: `sys` redefined here

Remove definition: `sys`

(F811)